### PR TITLE
Implement and test book request list

### PR DESCRIPTION
**Pull request by banguyenvareal**

### CODE REVIEW

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -124,3 +124,7 @@ a {
   font-size: 1.25rem;
 }
 
+.paginate {
```
Comment (dinhvareal) [inline app/assets/stylesheets/custom.scss]: delete it!

File: app/views/book_requests/index.html.erb [hunk 2]
```diff
@@ -6,8 +6,9 @@
           <img src="<%= book_request.book_request_images&.first.thumb_url %>
             "alt = "Book's cover" class="book-request-image">
           <div class="col-md-12 book-request-content" >
-            <p class="book-request-title"><%= book_request.name %></p>
-            <p class="book-request-detail">User: Obama</p>
+            <p class="book-request-title"> <%= truncate(book_request.name, length: 25) %></p>
+            <p class="book-request-detail"><%=t('book_request.buyer') %>
+              <%= User.find(book_request.user_id).name %></p>
```
Comment (dinhvareal) [inline app/views/book_requests/index.html.erb]: use delegate

File: app/assets/stylesheets/custom.scss [hunk 3]
```diff
@@ -124,3 +124,4 @@ a {
   font-size: 1.25rem;
 }
 
+
```
Comment (dinhvareal) [inline app/assets/stylesheets/custom.scss]: extra blank line end of file

File: app/views/book_requests/index.html.erb [hunk 4]
```diff
@@ -6,8 +6,9 @@
           <img src="<%= book_request.book_request_images&.first.thumb_url %>
             "alt = "Book's cover" class="book-request-image">
           <div class="col-md-12 book-request-content" >
-            <p class="book-request-title"><%= book_request.name %></p>
-            <p class="book-request-detail">User: Obama</p>
+            <p class="book-request-title"> <%= truncate(book_request.buyer_name, length: 25) %></p>
+            <p class="book-request-detail"><%=t('book_request.buyer') %>
+              <%= User.find(book_request.user_id).name %></p>
```
Comment (dinhvareal) [inline app/views/book_requests/index.html.erb]: remove

File: app/views/book_requests/index.html.erb [hunk 5]
```diff
@@ -6,8 +6,9 @@
           <img src="<%= book_request.book_request_images&.first.thumb_url %>
             "alt = "Book's cover" class="book-request-image">
           <div class="col-md-12 book-request-content" >
-            <p class="book-request-title"><%= book_request.name %></p>
```
Comment (dinhvareal) [inline app/views/book_requests/index.html.erb:9]: Why did you remove book request title

**Review by dinhvareal — CHANGES_REQUESTED** [review]

**Review by dinhvareal — APPROVED** [review]

**Review by mrcuong84 — APPROVED** [review]

**Review by hieunguyentrung — APPROVED** [review]

**Pull request merged**